README.md

```
@@ -1,126 +1,221 @@
+
+# 개발 서버 실행
+npm run dev
+```
+
+### 환경 변수 설정
+프로젝트 루트에 `.env.local` 파일을 생성하고 필요한 환경 변수를 설정하세요.
+
+
+## 📅 개발 로드맵
+
+### 완료된 작업
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)